src/gentropy/susie_finemapper.py

```diff
@@ -896,7 +898,63 @@ def susie_finemapper_one_sl_row_gathered_boundaries(  # noqa: C901
             logging.warning("Analysis Flags check failed for this study")
             return None
 
-        schema = StudyLocus.get_schema()
+        schema = StructType(
```

Comment:
@Daniel-Considine what is the difference between this schema and StudyLocus schema?

Reply:
It was breaking because the locus column from LocusBreaker (specifically `annotate_locus_statistics_boundaries`) has less columns than the StudyLocus schema.